frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -61,18 +59,11 @@ export const navigationLogic = kea<navigationLogicType>([
     })),
     reducers({
         // Non-mobile base
-        isSideBarShownBase: [
-            true,
-            { persist: true },
-            {
-                toggleSideBarBase: (state, { override }) => override ?? !state,
-            },
-        ],
+        isSideBarShownBase: [true, { persist: true }, {}],
```

Comment:
Can this be removed? Looks like it is not referenced anywhere in the codebase

Reply:
Yup, I'll remove it with rest of the sidebar code